### ISSUE TITLE
add serviceusage.googleapis.com api to essential_contacts example

### ISSUE THREAD

**Issue by apeabody** [terraform-google-modules/terraform-google-project-factory#718]
### TL;DR

add serviceusage.googleapis.com api to essential_contacts example

### Observed behavior

Error: Error enabling the Compute Engine API required to delete the default network: failed to send enable services request: googleapi: Error 403: Service Usage API has not been used in project [...] before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/serviceusage.googleapis.com/overview?project=[...] then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.

### Terraform Configuration

```hcl
CI
```

### Terraform Version

```sh
CI
```
